src/style.css

```diff
@@ -1 +1,70 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+:root {
+  --colorgrey: #e1e7ed;
+  --dropcolor: #00000040;
+  --fontcolor: #3d4e61;
+}
+
+.search {
+  margin-top: 20px;
+}
+
+.search__input {
+  width: 100%;
+  box-sizing: border-box;
+  background-image: url(/src/images/Search.svg);
+  background-repeat: no-repeat;
+  color: var(--fontcolor);
+  border: 1px solid var(--colorgrey);
+  border-radius: 4px;
+  outline: none;
+  box-shadow: 0 1px 8px #3d4e611a;
+  font-family: Avenir, sans-serif;
```

Comment:
It is considered best practices to apply the "default" font to the `body` tag so that every element that doesn't specify a font already uses it by default